### DISCUSSION TITLE
Github issue templates 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#45] [Q&A] [answered]
ToDo 작성에 있어서 좀 더 자동화를 하고 싶습니다..  

labels, assignees은 아래와 같이 템플릿에서 자동 등록이 가능한데 혹시 마일스톤, Project도 바로 등록이 가능한가요?

![image](https://user-images.githubusercontent.com/84510455/219044638-c1c6bd47-8a58-415f-b9fd-4c1e3b055a16.png)

++  - [ ] 선택 <- 눌러서 바로 이슈를 생성하는 것에도 자동으로 관련 사항을 자동으로 추가하는게 가능한가요?

**Comment by jongfeel** [comment, marked as answer]
github template은 스크린샷에 있는 기능 까지만 가능할겁니다.

저 같은 경우는 github cli인 gh를 좀 활용했었는데
github actions에서 쓰거나 아니면 local에서 gh를 활용해 template을 직접 만드는 방법도 고려해 볼 수 있을 것 같네요.

---

조금 더 검색해 보니 github actions를 쓰는 게 좋아 보이긴 합니다.
https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows

**Reply by fkdl0048** [reply]
그때 알려주신 gh를 사용하면 action단계에서 좀 더 자유롭게 가능하겠네요..!  

감사합니다

**Reply by fkdl0048** [reply]
++ 커피 잘 마시겠습니다!

**Reply by fkdl0048** [reply]
```
gh issue edit 159 --add-project "TodoProject"  
'TodoProject' not found
```
자꾸 찾을 수 없다는 문제가 발생해서 알아보니.. Beta라 지원하지 않는 것 같다고 하네요

https://github.com/cli/cli/issues/4547

**Reply by fkdl0048** [reply]
실제로 1달전에 병합된 기능이라고 하네요..!

https://github.com/cli/cli/issues/4547#issuecomment-1397687704

지금은 해결했습니다..

```
PS D:\ToDo> gh issue edit 159 --add-project "Todo"
https://github.com/fkdl0048/ToDo/issues/159
```

**Reply by fkdl0048** [reply]
![image](https://user-images.githubusercontent.com/84510455/219618396-64299fa7-f56d-401d-942b-15149412dc37.png)

성공 했습니다,,!
